fieldsolver/gridGlue.cpp

```diff
@@ -518,3 +518,76 @@ std::vector<CellID> mapDccrgIdToFsGridGlobalID(dccrg::Dccrg<SpatialCell,dccrg::C
    return fsgridIDs;
 }
 
+void feedBoundaryIntoFsGrid(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid,
```

Comment:
Since the boundarytype on fsrid depends on the resolution on the Vlasov grid, communication is needed for these to match. This way the type is determined in the Vlasov grid and fed directly into fsgrid, which then determines the layer in fsgrid.

Reply:
Righto.